hyperjumptech/monika
# Added: Install monika via Chocolatey

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Add additional github action to publish package to chocolatey so monika can be installed via chocolatey
2. update docs

## How did you implement / how did you fix it  
1.  Adding steps to `Upload binary` workflow specifically for windows OS

## How to test  
1. have a windows OS
2. open PowerShell in administrator mode
3. [install chocolatey](https://chocolatey.org/install)
![2](https://user-images.githubusercontent.com/5974862/157422234-e07676ee-5826-45e7-8810-8895da61bed1.PNG)

4. install monika with this command `choco install monika` and see monika version `monika -v` and see the monika version installed
![3](https://user-images.githubusercontent.com/5974862/157422329-223a66d2-e17d-4821-9393-361d2df74581.PNG)

5.  run `monika -c monika.yml` on directory where you put the monika.yml file
![4](https://user-images.githubusercontent.com/5974862/157422491-47d49100-d13d-45bc-922c-1fcf5eb3e91b.PNG)

## Additional Info
Example of success workflow github actions
https://github.com/raosan/monika/runs/5480701746?check_suite_focus=true
